genai-on-vertex-ai/genai-website-mod/backend/config.toml

```diff
@@ -12,13 +12,13 @@ recs_serving_config_id = ""
 [imagen]
 generation_model_name = "imagegeneration@005"
 caption_model_name = "imagetext@002"
-bucket_name = ""
+bucket_name = "kalschi-for-wsm-images"
```

Comment:
Let's remove this bucket names as well and point to the user provided bucket name

Reply:
removed